Index: check-0.9.5/configure.ac
===================================================================
--- check-0.9.5.orig/configure.ac
+++ check-0.9.5/configure.ac
@@ -27,7 +27,7 @@ AC_SUBST(CHECK_MICRO_VERSION)
 AC_SUBST(CHECK_VERSION)
 
 # Configure options.
-
+m4_ifdef([AM_PROG_AR], [AM_PROG_AR])
 AC_ARG_ENABLE(gcov,
 AC_HELP_STRING([--enable-gcov],
 	       [turn on test coverage @<:@default=no@:>@]),
